ui/snippets/topBar/DeFiDropdown.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button, Box, Flex, Popover, PopoverTrigger, PopoverContent, PopoverBody, useDisclosure } from '@chakra-ui/react';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+import { route } from 'nextjs-routes';
+
+import config from 'configs/app';
+import useIsMobile from 'lib/hooks/useIsMobile';
+import getPageType from 'lib/mixpanel/getPageType';
+import * as mixpanel from 'lib/mixpanel/index';
+import IconSvg from 'ui/shared/IconSvg';
+
+import DeFiDropdownItem from './DeFiDropdownItem';
+
+const feature = config.features.deFiDropdown;
+
+const DeFiDropdown = () => {
+  const router = useRouter();
+  const source = getPageType(router.pathname);
+  const isMobile = useIsMobile();
+  const { isOpen, onToggle, onClose } = useDisclosure();
+
+  const handleClick = React.useCallback((content: string) => {
+    mixpanel.logEvent(mixpanel.EventTypes.BUTTON_CLICK, { Content: content, Source: source });
+  }, [ source ]);
+
+  if (!feature.isEnabled) {
+    return null;
+  }
+
+  const buttonStyles = {
+    variant: 'solid',
+    size: 'xs',
+    borderRadius: 'sm',
+    height: 5,
+    px: 1.5,
+    fontWeight: '500',
+  };
+
+  const items = feature.items.map((item) => ({
+    ...item,
+    onClick: () => handleClick(item.text),
+  }));
+
+  return items.length > 1 ? (
+    <Popover isOpen={ isOpen } onClose={ onClose } placement="bottom-start" isLazy>
+      <PopoverTrigger>
+        <Button
+          onClick={ onToggle }
+          { ...buttonStyles }
+          bgColor={ isOpen ? 'blue.400' : undefined }
+          _active={{ bgColor: 'blue.400' }}
+        >
+          { !isMobile && 'Blockscout' } DeFi
```

Comment:
Let's use Show/Hide components or CSS styles, so the name does not flicker when the page is loading on mobile devices.